go/chat/context.go

```diff
@@ -57,7 +58,7 @@ func CtxIdentifyNotifier(ctx context.Context) *IdentifyNotifier {
 	return nil
 }
 
-func CtxAddLogTags(ctx context.Context) context.Context {
+func CtxAddLogTags(ctx context.Context, env *libkb.Env) context.Context {
```

Comment:
For this sort of thing, we usually abstract out the relevant features of something like `libkb.Env` into an interface.  Does that make sense here?